src/index.js

```diff
@@ -84,3 +86,13 @@ Promise.all(
   localStorage.setItem("contributors", JSON.stringify(allContributors));
   generateCardGrid(allContributors);
 });
+
+const searchHacker = (e) => {
+  const filteredContributors = allContributors.filter((contrib) => {
+    const name = contrib.name || contrib.login;
+    return name.toLowerCase().includes(e.target.value.toLowerCase());
```

Comment:
Yup, but if someone wants to search via username this does not help. For example searching for [`@Dexters-Hub`](https://github.com/Dexters-Hub) does not show `Alwin John`.

So maybe we do something like:

```js
const usr_name = contrib.login
const name = contrib.name || usr_name
return name.toLowerCase().includes(e.target.value.toLowerCase()) || usr_name.toLowerCase().includes(e.target.value.toLowerCase())
```

Reply:
Looks good to me. We can do like this instead.
```js
const usr_name = contrib.login
- const name = contrib.name || usr_name
+ const name = contrib.name || "hacker"
return name.toLowerCase().includes(e.target.value.toLowerCase()) || usr_name.toLowerCase().includes(e.target.value.toLowerCase())
```

If I search "ab", I am getting two hackers there. It might be confusing for others.

Reply:
I think it would be useful to search for "hacker" too. It is difficult to filter them. It also gets confusing when I search for "ab" and get two "hackers" in the results. What do you think of this change?

```js
const usr_name = contrib.login
- const name = contrib.name || usr_name
+ const name = contrib.name || "hacker"
return name.toLowerCase().includes(e.target.value.toLowerCase()) || usr_name.toLowerCase().includes(e.target.value.toLowerCase())
```

Reply:
You're showing this diff right?

```diff
const usr_name = contrib.login
- const name = contrib.name || usr_name
+ const name = contrib.name || "hacker"
return name.toLowerCase().includes(e.target.value.toLowerCase()) || usr_name.toLowerCase().includes(e.target.value.toLowerCase())
```

I think that's fine 'cause there could be more than one `login` with same `name`. Aside from the repo/description/follower info, hovering over the card also shows the link to their profile.

Reply:
By the way, searching for the word "hacker" does not show both the users ( `https://github.com/CHAITHANYA-BABU` and `https://github.com/abilash-sajeev`)

Reply:
A new user to the site may not realize that "hacker" is just a hardcoded string we gave them since their name is `null`. I guess it would be a better user experience to filter them just like other names.

Reply:
Could you please make those changes and push a commit?

Reply:
I've to go now, will do asap.